src/lib/rest/rest.cpp

```diff
@@ -240,7 +240,7 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
   {
     containsForbiddenChars = forbiddenChars(val, ";");
   }
-  else if ((key != "q") && (key != "idPattern"))
+  else if ((key != "q") && (key != "mq") && (key != "idPattern"))
```

Comment:
I agree. I'd suggest to create a new issue (techdebt nice2have about this) so we don't forgot it in the future.